src/main/archetype/ui.content/src/main/content/jcr_root/content/__appId__/.content.xml

```diff
@@ -6,6 +6,7 @@
         cq:conf="/conf/${appId}"
         jcr:primaryType="cq:PageContent"
         jcr:title="${appTitle}"
+        sling:configRef="/conf/${appId}"
```

Comment:
![image](https://user-images.githubusercontent.com/689884/89764655-1ad17a00-daf5-11ea-891f-137eb0167429.png)

anonymous access on publish is not possible for the `sling:conf` folder. Please add ACL entry with `rep:glob` on `/conf` accordingly.

Reply:
I already added access to `/conf/_appId_/_sling_configs` in https://github.com/adobe/aem-project-archetype/pull/448/files#diff-5f93dfb9442aa73184a52dae218f20a1

Reply:
Sorry for the noise. The issue was just an outdated core bundle.